datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/auth/AccessTokenUtil.java

```diff
@@ -3,27 +3,30 @@
 import com.linkedin.datahub.graphql.generated.AccessTokenDuration;
 import java.time.Duration;
 import java.time.temporal.ChronoUnit;
+import java.util.Optional;
 
 
 public class AccessTokenUtil {
 
   /**
    * Convert an {@link AccessTokenDuration} into its milliseconds equivalent.
    */
-  public static long mapDurationToMs(final AccessTokenDuration duration) {
```

Comment:
What use do we have of keeping this Optional? Since we want to make use of annotations to indicate nulls, let's keep it consistent throughout the PR. It will further reduce the changes of this PR overall.

Reply:
+1 in general I agree with using annotation over boxing with Optional.